test/livebook_web/live/session_live_test.exs

```diff
@@ -2632,4 +2632,54 @@ defmodule LivebookWeb.SessionLiveTest do
       assert File.read!(dockerfile_path) =~ "COPY notebook.livemd /apps"
     end
   end
+
+  test "defined modules under sections", %{conn: conn, session: session} do
+    Code.put_compiler_option(:debug_info, true)
+
+    Session.subscribe(session.id)
+
+    {:ok, view, _} = live(conn, ~p"/sessions/#{session.id}")
+    refute render(view) =~ "MyBigModuleName"
+
+    cell_id =
+      insert_text_cell(session.pid, insert_section(session.pid), :code, ~S'''
+      defmodule MyBigModuleName do
```

Comment:
We use embedded runtime here, which means the modules are defined in the current node, so let's use more specific names, like `SessionLiveTest.MyModule`.